client/src/index.scss

```diff
@@ -0,0 +1,98 @@
+@import url('https://fonts.googleapis.com/css2?family=Lora:wght@400;500;600;700&family=Oswald:wght@200;300;400;500;600;700&display=swap');
```

Comment:
we already have an index.css file.
did you take it into consideration when making this index.scss file?

Reply:
I replaced the index.css with index.scss file :) we don't have both files.